.travis.yml

```diff
@@ -1,10 +1,13 @@
 language: node_js
 node_js:
   - 4
```

Comment:
Node.js v4 has reached its end-of-life.
Should we remove 4 and add 9 and 10?
https://github.com/nodejs/Release